lib/cunit/cu_pc_patch_lazperf.c

```diff
@@ -246,6 +246,41 @@ test_wkb_lazperf()
 	pcfree(wkb1);
 	pcfree(wkb2);
 }
+
+static void
+test_patch_filter_lazperf()
```

Comment:
I'd suggest using a more explicit name for the test function. Like `test_patch_filter_lazperf_zero_points`.